File: csharp/pangram/Pangram.cs
using System;
using System.Collections.Generic;

public static class Pangram
{
    public static bool IsPangram(string input)
    {
        var data = new List<char>("abcdefghijklmnopqrstuvwxyz");

        foreach (char c in input.ToLower())
        {
            data.Remove(c);
            if (data.Count == 0) { break; }
        }

        return data.Count == 0;
    }
}

File: go/pangram/pangram.go
package pangram

import (
	"unicode"
)

// Determine if a string contains all letters of the alphabet
func IsPangram(input string) bool {

	set := make(map[rune]bool)

	for _, item := range input {

		item = unicode.ToLower(item)

		if !unicode.IsLetter(item) {
			continue
		}

		if _, ok := set[item]; !ok {
			set[item] = true
		}
	}

	return len(set) == 26
}
